src/databricks/labs/ucx/assessment/aws.py

```diff
@@ -455,3 +533,68 @@ def _run_command(self, command: str):
     @staticmethod
     def _get_json_for_cli(input_json: dict) -> str:
         return json.dumps(input_json).replace('\n', '').replace(" ", "")
+
+
+@dataclass()
+class AWSGlue:
+    aws_region: str
+    aws_account_id: str
+
+    @classmethod
+    def get_glue_connection_info(
+        cls,
+        workspace: WorkspaceClient,
```

Comment:
I recommend to be more restrictive with passing the workspace client to such a (relatively) low level object as the workspace client is a HUGE object with many entrypoints therefore hard to see why it is needed in this case. You could also expect the metastore (region) instead